scraper/src/Scraper.hs

```diff
@@ -22,7 +23,45 @@ runScrape = do
 scrape :: IO ()
 scrape = do 
   website <- httpLBS "http://leg.colorado.gov/bill-search?field_sessions=10171&sort_bef_combine=field_bill_number%20ASC" 
-  L8.writeFile "test.html" $ getResponseBody website
+  L8.writeFile "test/example/bill_page_1.html" $ getResponseBody website
```

Comment:
You don't want to rely on having a hard drive on this service. `writeFile` is useful for acquiring test data locally, but you'll want to keep the data in memory in production.

Instead of writing the file directly to disk, just return the file. Then, other functions can use the website contents without having to read a file.

Reply:
Fixed.